File: legacy/bankcard-master/backend/src/main/java/gov/nist/oism/asd/empbc/model/PcItem.java
package gov.nist.oism.asd.empbc.model;

import com.fasterxml.jackson.annotation.JsonFormat;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import java.io.Serializable;
import java.util.Date;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@JsonIgnoreProperties(ignoreUnknown = true)
public class PcItem implements Serializable {

    private static final long serialVersionUID = 1L;

    private Integer requestId;
    private Integer fy;
    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSX")
    private Date createdDate;
    private String requisitionNumber;
    private Integer ouId;
    private Integer divId;
    private Integer grpId;
    private String ou;
    private String division;
    private String group;
    private String vendor;
    private Integer itemId;
    private String catelogNumber;
    private String itemName;
    private String itemDescription;
    private Double price;
    private Integer quantity;
    private String purpose;
    private String isChemical;
    private Integer shoppingCartFileId;
    private String itemStatus;
    private Integer itemStatusId;
    private String projectTask;
    private String objectClass;
    private String isTaggableEquipment;
    private Double priceOrdered;
    private Integer quantityOrdered;
    private String itemNotes;
    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSX")
    private Date dateReceived;
    private String transactionNumber;
    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSX")
    private Date statementDate;
    private String unitIssue;
    private Integer purchaseTypeId;
}

File: legacy/bankcard-master/frontend/app/model/PcItem.js
/**
 * for Property Custodian report
 */


Ext.define('bcp.model.PcItem', {
    extend: 'Ext.data.Model',
    alias: 'model.pcitem',

    requires: [
        'Ext.data.field.String',
        'Ext.data.field.Number',
        'Ext.data.field.Boolean',
        'Ext.data.field.Date'
    ],

    idProperty: 'itemId',

    fields: [
        { type: 'int', name: 'requestId' },
        { type: 'int', name: 'fy' },
        { type: 'date', name: 'createdDate' },
        { type: 'string', name: 'requisitionNumber' },
        { type: 'int', name: 'ouId' },
        { type: 'int', name: 'divId' },
        { type: 'int', name: 'grpId' },
        { type: 'string', name: 'ou' },
        { type: 'string', name: 'division' },
        { type: 'string', name: 'group' },
        { type: 'string', name: 'vendor' },
        { type: 'int', name: 'itemId', },
        { type: 'string', name: 'catelogNumber' },
        { type: 'string', name: 'itemName' },
        { type: 'string', name: 'itemDescription' },
        { type: 'float', name: 'price' },
        { type: 'int', name: 'quantity' },
        { type: 'string', name: 'purpose'},
        { type: 'string', name: 'isChemical' },
        { type: 'int', name: 'shoppingCartFileId' },
        { type: 'string', name: 'itemStatus' },
        { type: 'int', name: 'itemStatusId' },
        { type: 'string', name: 'projectTask' },
        { type: 'string', name: 'objectClass' },
        { type: 'string', name: 'isTaggableEquipment' },
        { type: 'float', name: 'priceOrdered' },
        { type: 'int', name: 'quantityOrdered' },
        { type: 'string', name: 'itemNotes' },
        { type: 'date', name: 'dateReceived' },
        { type: 'string', name: 'transactionNumber' },
        { type: 'date', name: 'statementDate' },
        { type: 'string', name: 'unitIssue' },
        { type: 'int', name: 'purchaseTypeId' }
    ]
});
